src/plugin.js

```diff
@@ -217,7 +220,16 @@ const contribAdsPlugin = function(options) {
   // But first, cast to boolean.
   settings.stitchedAds = !!settings.stitchedAds;
 
-  if (settings.stitchedAds) {
+  if (settings.playerMode === 'outstream') {
```

Comment:
I'm not sure where the best place to set the player's src in contrib-ads is. I was thinking either here in the plugin initialisation or in the first Outstream state (OutstreamPending(

Reply:
Instead of keeping the full string here, it might be nice to start a constants file and import it. Just to keep things a bit more concise and legible.

Reply:
Also, to your point: since we're setting player instance listeners and parameters in this general block of code, I think it's fine to set the src here to consolidate if statements. I'd just add a comment for posterity.

Reply:
Good point! Will create a constants file and put the video file in there :)